services/schedulable_transaction_body.proto

```diff
@@ -78,269 +91,333 @@ import "node_update.proto";
-
-    /**
-     * Generates a pseudorandom number.
-     */
-    UtilPrngTransactionBody util_prng = 40;
-
-    /**
-     * Update the metadata of one or more NFT's of a specific token type.
-     */
-    TokenUpdateNftsTransactionBody token_update_nfts = 41;
-
-    /**
-     * Transaction body for a scheduled transaction to create a new node.
-     */
-    com.hedera.hapi.node.addressbook.NodeCreateTransactionBody nodeCreate = 42;
-
-    /**
-     * Transaction body for a scheduled transaction to modify an existing node.
-     */
-    com.hedera.hapi.node.addressbook.NodeUpdateTransactionBody nodeUpdate = 43;
-
-    /**
-     * Transaction body for a scheduled transaction to remove a node.
-     */
-    com.hedera.hapi.node.addressbook.NodeDeleteTransactionBody nodeDelete = 44;
-
-    /**
-     * A transaction body to "reject" undesired tokens.<br/>
-     * This transaction will transfer one or more tokens or token
-     * balances held by the requesting account to the treasury
-     * for each token type.
+     * A limit for the transaction fee the client is willing to pay.
      * <p>
-     * Each transfer MUST be one of the following:
-     * <ul>
-     *   <li>A single non-fungible/unique token.</li>
-     *   <li>The full balance held for a fungible/common
-     *       token type.</li>
-     * </ul>
-     * When complete, the requesting account SHALL NOT hold the
-     * rejected tokens.<br/>
-     * Custom fees and royalties defined for the tokens rejected
-     * SHALL NOT be charged for this transaction.
+     * The network SHALL NOT charge fees greater than this value.
      */
-    TokenRejectTransactionBody tokenReject = 45;
+    uint64 transactionFee = 1;
 
     /**
-     * Transaction body for a scheduled transaction to cancel an airdrop.
-     */
-    TokenCancelAirdropTransactionBody tokenCancelAirdrop = 46;
-
-    /**
-     * Transaction body for a scheduled transaction to claim an airdrop.
-     */
-    TokenClaimAirdropTransactionBody tokenClaimAirdrop = 47;
-
-    /**
-     * Transaction body for a scheduled transaction to airdrop tokens.
-     */
-    TokenAirdropTransactionBody tokenAirdrop = 48;
-  }
+     * A short description of the schedulable transaction.
+     * <p>
+     * This value, if set, MUST NOT exceed 100 bytes when encoded as UTF-8.
+     */
+    string memo = 2;
+
+    // The choices here are arranged by service in roughly lexicographic
+    // order. The field ordinals are non-sequential, and a result of the
+    // historical order of implementation.
+    oneof data {
+        /**
+         * Create a topic.
+         */
+        ConsensusCreateTopicTransactionBody consensusCreateTopic = 18;
+
+        /**
+         * Update a topic.
+         */
+        ConsensusUpdateTopicTransactionBody consensusUpdateTopic = 19;
+
+        /**
+         * Delete a topic.
+         */
+        ConsensusDeleteTopicTransactionBody consensusDeleteTopic = 20;
+
+        /**
+         * Submit a message to a topic.<br/>
```

Comment:
In the Style Guidelines it states "Include a line break `(<br/>)` after the first line, then a brief description, if appropriate." Is this applicable to fields as well?

Reply:
Added some clarification to the guidelines.